chainer/initializer.py

```diff
@@ -16,6 +40,9 @@ def __call__(self, array):
             array (numpy.ndarray or cupy.ndarray):
                 An array to be initialized by this initializer.
 
+        Returns:
+            numpy.ndarray or cupy.ndarray: An initialized array.
```

Comment:
Does the call operator returns the array? It seems the overridden implementations do not return anything.